### PR TITLE
Add LSM6DS3 Gyroscope implementation

### PR DESCRIPTION
**Pull request by krishnachaitanya7**
- Added LSM6DS3 Gyroscope implementation

### CODE REVIEW

File: robotic_skin/sensor/lsm6ds3.py [hunk 1]
```diff
@@ -10,7 +10,7 @@
 from robotic_skin.const import GRAVITATIONAL_CONSTANT
```
Comment (watakandai) [inline robotic_skin/sensor/lsm6ds3.py:10]: We might need to change the gravity constant from `9.819` to `9.8`

Comment (watakandai) [reply robotic_skin/sensor/lsm6ds3.py:10]: since we are using 9.8 in calibration.

Comment (krishnachaitanya7) [reply robotic_skin/sensor/lsm6ds3.py:10]: So shall I change 9.819 to 9.8 in const.py? @watakandai

Comment (watakandai) [reply robotic_skin/sensor/lsm6ds3.py:10]: I'll change the gazebo setting to use 9.819, so let it be as it is

**Review by watakandai — APPROVED** [review]

**Pull request merged**